ci/requirements/linux_website_deps_pip.txt

```diff
@@ -6,7 +6,7 @@
  sphinx_gallery
  sphinxcontrib-apidoc
  myst-parser
- pydata-sphinx-theme
+ pydata-sphinx-theme!=0.10
```

Comment:
Looks like this still matches against 0.10.1:

```
 Collecting pydata-sphinx-theme!=0.10
  Downloading pydata_sphinx_theme-0.10.1-py3-none-any.whl (1.5 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.5/1.5 MB 129.4 MB/s eta 0:00:00
```

Any idea what magic comparison operator has to be used to not get 0.10.x?

Reply:
But that won't prevent ending up with 0.10 if other constraints in their enviroment cause it.